src/main/java/org/interledger/ilp/client/LedgerClient.java

```diff
@@ -156,6 +159,36 @@ public UUID sendTransfer(InterledgerAddress to, MonetaryAmount amount,
     return id;
   }
 
+  public void fulfillTransfer(UUID transferId, Fulfillment fulfillment) {
+    this.adaptor.fulfillTransfer(transferId, fulfillment);
+  }
+  
+  public UUID makePayment(QuoteResponse quote, InterledgerPaymentRequest paymentRequest) {
+    
+    //TODO Make sure that the quote matches the IPR
+    
+    byte[] data = generateTransferMemo(paymentRequest);
+    ZonedDateTime expiresAt = calculateLocalExpiry(paymentRequest.getExpiresAt());
+    
+    return sendTransfer(
+        quote.getSourceConnectorAccount(), 
+        quote.getSourceAmount(), 
+        paymentRequest.getCondition(), 
+        expiresAt, 
+        data);
+        
+  }
+  
+  private byte[] generateTransferMemo(InterledgerPaymentRequest paymentRequest) {
+    // TODO Auto-generated method stub
+    return null;
+  }
+
+  private ZonedDateTime calculateLocalExpiry(ZonedDateTime expiresAt) {
```

Comment:
Im not quite sure what the point of this is - if the payment request already has an expiry in some time zone, why not simply pass that along, surely we dont want to convert into local times for each hop? Or is the point to add on some additional time?